### PR TITLE
delete original description of preventExtensions of english in chinese

### PR DESCRIPTION
**Pull request by songbing97**
delete original description of english

### Description
image

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods/index.md [hunk 1]
```diff
@@ -23,6 +23,6 @@ ES2015 中引入的 {{jsxref("Reflect")}} 对象是一个内置对象，提供
...
 | `isExtensible()`              | 如果对象是可扩展的，则 Object.isExtensible（）返回 true，否则返回 false。如果第一个参数不是对象（原始值），则在 ES5 中抛出`TypeError`。在 ES2015 中，它将被强制为不可扩展的普通对象并返回`false`。  | 如果对象是可扩展的，则{{jsxref("Reflect.isExtensible()")}} 返回`true`，否则返回`false`。如果第一个参数不是对象（原始值），则抛出`TypeError`。                                                                                                                                                                 |
-| `preventExtensions()`         | {{jsxref("Object.preventExtensions()")}} 返回被设为不可扩展的对象。如果参数不是对象（原始值），则在 ES5 中抛出`TypeError`。在 ES2015 中，参数如为不可扩展的普通对象，然后返回对象本身。 | returns `true` if the object has been made non-extensible, and `false` if it has not. Throws a `TypeError` if the argument is not an object (a primitive).如果对象已变得不可扩展，则{{jsxref("Reflect.preventExtensions()")}} 返回`true`，否则返回`false`。如果参数不是对象（原始值），则抛出`TypeError`。 |
+| `preventExtensions()`         | {{jsxref("Object.preventExtensions()")}} 返回被设为不可扩展的对象。如果参数不是对象（原始值），则在 ES5 中抛出`TypeError`。在 ES2015 中，参数如为不可扩展的普通对象，然后返回对象本身。 | 如果对象已变得不可扩展，则{{jsxref("Reflect.preventExtensions()")}} 返回`true`，否则返回`false`。如果参数不是对象（原始值），则抛出`TypeError`。 |
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods/index.md:26]:
```suggestion
| `preventExtensions()`         | {{jsxref("Object.preventExtensions()")}} 返回被设为不可扩展的对象。如果参数不是对象（为原始值），则在 ES5 中抛出 `TypeError`。在 ES2015 中，则将参数视为不可扩展的普通对象，并返回对象本身。 | 如果对象已变得不可扩展，则 {{jsxref("Reflect.preventExtensions()")}} 返回 `true`，否则返回 `false`。如果参数不是对象（为原始值），则抛出 `TypeError`。 |
```

**Review by yin1999 — COMMENTED** [review]
A suggestion

**Review by yin1999 — APPROVED** [review]
Thanks a lot. Welcome to the community :tada:

**Comment by songbing97** [comment]
> Thanks a lot. Welcome to the community 🎉

First successful PR in my career, thank you, dei佬。

**Pull request merged**